src/common/commonutils/PassUtils.c

```diff
@@ -147,23 +147,102 @@ int SetLockoutForFailedPasswordAttempts(void* log)
     const char* etcPamdLogin = "/etc/pam.d/login";
     const char* etcPamdSystemAuth = "/etc/pam.d/system-auth"; 
     const char* etcPamdPasswordAuth = "/etc/pam.d/password-auth";
+    const char* etcPamdLoginCopy = "/etc/pam.d/~login.copy";
+    const char* etcPamdSystemAuthCopy = "/etc/pam.d/~system-auth.copy";
+    const char* etcPamdPasswordAuthCopy = "/etc/pam.d/~password-auth.copy";
     const char* marker = "auth";
-
-    int status = ENOENT;
+    char* original = NULL;
+    int status = ENOENT, _status = ENOENT;
```

Comment:
Quick check: If no file exists for etcPamdSystemAuth, we expect to return ENOENT?

Reply:
Correct. If none of the 3 files exist, the default status value of ENOENT is returned.

Reply:
And.... taking this back. None of these files found, the set shall succeed. Fixing this. Thanks for the feedback!